# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsFrameList.cpp
# Commit: 58733fee05e0
# Full Hash: 58733fee05e09a34891f992e7c71051e6581efbe
# Author: Ting-Yu Lin <tlin@mozilla.com>
# Date: 2022-10-12 09:45:09
# Regressor Bug: 1794456
# File Overlap Count: 1
# Description:
#   Bug 1794456 Part 2 - Remove DEBUG members in nsFrameList::Slice and nsFrameList::Enumerator. r=emilio
#   
#   They are set but never used.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D158985
# ==============================================================================

diff -r 7f0b494afb59 -r 58733fee05e0 layout/generic/nsFrameList.cpp
--- a/layout/generic/nsFrameList.cpp	Tue Oct 11 21:01:27 2022 +0000
+++ b/layout/generic/nsFrameList.cpp	Tue Oct 11 21:01:27 2022 +0000
@@ -158,7 +158,7 @@
   VerifyList();
 
   aFrameList.Clear();
-  return Slice(*this, firstNewFrame, nextSibling);
+  return Slice(firstNewFrame, nextSibling);
 }
 
 nsFrameList nsFrameList::TakeFramesBefore(nsIFrame* aFrame) {